deploy

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+cp prod.env .env 
+docker-compose -f docker-compose.yml -f docker-compose.prod.yml down --remove-orphans
+docker-compose -f docker-compose.frontend.yml up --build --force-recreate
+cp -a frontend/build/. api/public/
+docker-compose -f docker-compose.yml -f docker-compose.prod.yml up --build --force-recreate -d
+docker-compose exec api sh -c "rails db:migrate RAILS_ENV=production"
```

Comment:
Actually, you can shorten this last command by calling `docker-compose exec api rails db:migrate RAILS_ENV=production` instead.